### PR TITLE
update Contribute

### PR DESCRIPTION
**Pull request by kuailehaibin**
> + update Contribute in Chinese
> + add Contribute in English

### CODE REVIEW

File: doc/us-en/about/How-to-contribute.md [hunk 1]
```diff
@@ -1,11 +1,54 @@
...
+
+OpenEdge use this [Git branching model](https://nvie.com/posts/a-successful-git-branching-model/). The following steps guide usual contributions.
+
+1. Fork
+   
+   Our development community has been growing fastly, so we encourage developers to submit code. And please file Pull Requests from your fork. To make a fork, please refer to Github page and click on the ["Fork" button](https://help.github.com/articles/fork-a-repo/). 
+
+2. Prepare for the development environment
+
+   If you want to contribute code to OpenEdge, please run:
```
Comment (ludanfeng) [inline doc/us-en/about/How-to-contribute.md]: 不如删了，每条命令都已经由注释

File: doc/us-en/about/How-to-contribute.md [hunk 2]
```diff
@@ -1,11 +1,54 @@
...
+   ```bash
+   go get github.com/baidu/openedge # clone openedge official repo
+   cd $GOPATH/github.com/baidu/openedge # step into openedge
+   git checkout master  # verify master branch
+   git remote add fork https://github.com/<your_github_account>/openedge  # specify remote repo
+   ```
+
+3. Push changes to your forked repo
+
+   Here, submit the changed requirements or fixed bugs to the remote repo of **fork** in step 2. For details, please run:
```
Comment (ludanfeng) [inline doc/us-en/about/How-to-contribute.md]: 不如删了，每条命令都已经由注释

File: doc/us-en/about/How-to-contribute.md [hunk 3]
```diff
@@ -1,11 +1,54 @@
...
+   ```bash
+   git status   # view current code change status
+   git add .
```
Comment (ludanfeng) [inline doc/us-en/about/How-to-contribute.md]: git add . # add all local changes

File: doc/us-en/about/How-to-contribute.md [hunk 4]
```diff
@@ -1,11 +1,54 @@
...
+   git commit -c "modify description"  # submit the code to the local repo with the code change description
```
Comment (ludanfeng) [inline doc/us-en/about/How-to-contribute.md]: commit changes with comment

File: doc/us-en/about/How-to-contribute.md [hunk 5]
```diff
@@ -1,11 +1,54 @@
...
+   git push fork # push changed code to remote repo
```
Comment (ludanfeng) [inline doc/us-en/about/How-to-contribute.md]: push code changes to remote repo which specifies your forked repository

File: doc/us-en/about/How-to-contribute.md [hunk 6]
```diff
@@ -1,11 +1,54 @@
...
+   ```
+
+4. Create pull request
+
+   You can push and file a pull request to OpenEdge official repo [https://github.com/baidu/openedge](https://github.com/baidu/openedge). To create a pull request, please follow [these steps](https://help.github.com/articles/creating-a-pull-request/). Once the OpenEdge code repo reviewer has passed your code submission and merge your request, you will see the code which you contributed in the OpenEdge official repo.
```
Comment (ludanfeng) [inline doc/us-en/about/How-to-contribute.md]: repo --> repository

Comment (ludanfeng) [reply doc/us-en/about/How-to-contribute.md]: Once the OpenEdge repository reviewer approves and merges your pull request, you will see the code which contributed by you in the OpenEdge official repository.

File: doc/us-en/about/How-to-contribute.md (hunk 1 above)
Comment (kuailehaibin) [reply doc/us-en/about/How-to-contribute.md]: 这不太好吧，直接放置命令

File: doc/us-en/about/How-to-contribute.md (hunk 3 above)
Comment (kuailehaibin) [reply doc/us-en/about/How-to-contribute.md]: 好的

File: doc/us-en/about/How-to-contribute.md (hunk 2 above)
Comment (kuailehaibin) [reply doc/us-en/about/How-to-contribute.md]: 行吧，删掉

File: doc/us-en/about/How-to-contribute.md (hunk 4 above)
Comment (kuailehaibin) [reply doc/us-en/about/How-to-contribute.md]: 好的

File: doc/us-en/about/How-to-contribute.md (hunk 5 above)
Comment (kuailehaibin) [reply doc/us-en/about/How-to-contribute.md]: 好的

File: doc/us-en/about/How-to-contribute.md (hunk 6 above)
Comment (kuailehaibin) [reply doc/us-en/about/How-to-contribute.md]: 好的

**Pull request merged**